tests/integration/notifications/test-class-yoast-notification-center.php

```diff
@@ -841,13 +841,15 @@ public function test_add_notifications_only_once_for_user() {
 
 		$instance = $this->get_notification_center();
 
-		$user_mock = $this->mock_wp_user( 3, [ 'wpseo_manage_options' => true ] );
+		$user_id = $this->factory->user->create();
```

Comment:
I think there is a `create_and_get` you can use. If you chose to apply, please also do so in the other tests 😉

Reply:
Done 👍